app/static/js/categories_filter.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener("DOMContentLoaded", function () {
+  document.getElementById("category-button").addEventListener("click", function () {
+    filterByCategory();
+  });
+});
+
+function filterByCategory(){
+    // TODO(issue#67): Allow filter by category name
+    const category = document.getElementById("category").value;
+
+    const css = (!Number.isInteger(+category) || !category || 0 === category.length) ? `
+    .event_line {
+      display: grid;
+      visibility: visible;
+    }
+    ` : `
+    .event_line {
+      display: none;
+      visibility: hidden;
+    }
+    .event_line[data-name*="${category}" i] {
+      display: grid;
+      visibility: visible;
+    }
+    `;
+    window.cssFilter.innerHTML = css;
+
+    const allEvents = document.getElementsByClassName("event_line");
+    for (event of allEvents) {
+      event.parentNode.style.display = "none" ;
```

Comment:
Maybe replace with CSS and dataset?

Reply:
Not sure what do you mean here?

Reply:
Same as above

Reply:
It's actually same as above - here I change the parent of "event line" element, which is "wrapper" element. 
This is done in order to hide titles of dates when all events in this day are filtered out.

If you mean something else, could you please elaborate?

Reply:
Changed as requested, done.